WordPress/gutenberg
# Fix Snapshots: Update color-palette and toggle-group snapshots with data-command

## What?

`npm run test:unit` is asking to update the snapshots
I'm not 100% sure if this data attribute is needed on these elements, so I'm asking for the review to components experts :-)

## Testing Instructions

Run `npm run test:unit` in trunk to check the errors.
Run `npm run test:unit` in the branch to check that are fixed.

### Testing Instructions for Keyboard


## Screenshots or screencast
